docs/create-manifest.md

```diff
@@ -0,0 +1,29 @@
+# CreateManifest
+
+The following console commands are available:
+
+- `vendor/bin/app-sdk manifest:add`
+
+## Adding an Manifest file
+
+The `vendor/bin/app-sdk manifest:add` adds a minimal Manifest file.
+
+### Arguments and Options
+
+#### Arguments
+
+- `name`
+
+`vendor/bin/app-sdk manifest:add "Your Manifest name"` will set the name of your Manifest file.
+
+
+- `local`
+
+`vendor/bin/app-sdk manifest:add "Your Manifest name" Locale ` will set the name and locale of your Manifest file.
+
+```
```

Comment:
WHat is this?

Reply:
It is for manifest name(set value for  'name' parameter into manifest file) and locale name(like: de_DE). if you don't enter Locale then by default it set en_US.